### PR TITLE
MODFEE-19 Implement waive notices, add tests

### PR DESCRIPTION
**Pull request by alexanderkurash**
Resolves [MODFEE-19](https://issues.folio.org/browse/MODFEE-19):
- Send patron notice when fee/fine charge is waived. 
- Rename template token `fee.actionAdditionalInfo` -> `fee.actionAdditionalInfoForPatron`.

### CODE REVIEW

**Comment by marcjohnson-kint** [comment]
@alexanderkurash Please can you help me understand why you've requested a review from me for this change? Are you only looking for my feedback?

(mod-feefines does not have a review policy, so no review is required)

**Comment by alexanderkurash** [comment]
> @alexanderkurash Please can you help me understand why you've requested a review from me for this change? Are you only looking for my feedback?
> 
> (mod-feefines does not have a review policy, so no review is required)

I've seen you approving one of the previous PRs. But you're right - if there's no review policy we'll handle it within Vega team.

**Comment by marcjohnson-kint** [comment]
> > @alexanderkurash Please can you help me understand why you've requested a review from me for this change? Are you only looking for my feedback?
> > (mod-feefines does not have a review policy, so no review is required)
> 
> I've seen you approving one of the previous PRs. But you're right - if there's no review policy we'll handle it within Vega team.

That was only because someone asked me to specifically (which I'm happy to do). I only wanted to clarify that it isn't necessary in this case.

File: src/main/java/org/folio/rest/domain/FeeFineNoticeContext.java [hunk 1]
```diff
@@ -44,7 +46,9 @@ private boolean isCharge() {
 
   private boolean isFeeFineActon() {
     return feefineaction.getTypeAction().equals(PAID_FULLY) ||
-      feefineaction.getTypeAction().equals(PAID_PARTIALLY);
+      feefineaction.getTypeAction().equals(PAID_PARTIALLY) ||
+      feefineaction.getTypeAction().equals(WAIVED_FULLY) ||
+      feefineaction.getTypeAction().equals(WAIVED_PARTIALLY);
```
Comment (OleksandrVidinieiev) [inline src/main/java/org/folio/rest/domain/FeeFineNoticeContext.java]: Maybe we should introduce a constant list of all fee/fine actions and replace this method's body with simple `List#contains` invocation? Keep in mind that this list is only going to grow in subsequent tasks.

Comment (alexanderkurash) [reply src/main/java/org/folio/rest/domain/FeeFineNoticeContext.java]: Done

**Pull request merged**